test/e2e-test/rollout_plan_test.go

```diff
@@ -426,7 +426,7 @@ var _ = Describe("Cloneset based rollout tests", func() {
 				appRollout.Spec.TargetAppRevisionName = utils.ConstructRevisionName(app.GetName(), 2)
 				appRollout.Spec.RolloutPlan.BatchPartition = nil
 				return k8sClient.Update(ctx, &appRollout)
-			}, time.Second*5, time.Millisecond*500).Should(Succeed())
+			}, time.Second*5, time.Millisecond).Should(Succeed())
```

Comment:
two frequency? `time.Second`?

Reply:
one of the e2e failed because the state has passed the "rolling" before the check below...